go.mod

```diff
@@ -91,4 +90,5 @@ require (
 )
 
 // replace github.com/gophercloud/gophercloud => ./external/gophercloud
-replace github.com/gophercloud/gophercloud => github.com/Wind-River/gophercloud v0.0.0-20230731150227-137664ed2caf
+// replace github.com/gophercloud/gophercloud => github.com/Wind-River/gophercloud v0.0.0-20230605171524-742ad279b1e1
+replace github.com/gophercloud/gophercloud => github.com/eliud-kyale/gophercloud v0.0.0-20230803204716-7896558c268e
```

Comment:
Need to switch back to the windriver repo after the commit in the gophercloud merged